client/js/transports/small-webrtc.mdx

```diff
@@ -0,0 +1,160 @@
+
+const transport = new SmallWebRTCTransport();
+
+const rtviClient = new RTVIClient({
+  transport,
+  enableCam: false, // Default camera off
+  enableMic: true, // Default microphone on
+  callbacks: {
+    // Event handlers
+  },
+  params: {
+    baseUrl: "http://localhost:7860",
+    endpoints: {
+      connect: "/api/offer",
+    },
+  },
+});
+
+await rtviClient.connect();
+```
+
+## API Reference
+
+### Constructor
+
+```typescript
+new SmallWebRTCTransport();
+```
+
+No parameters are required for basic functionality.
+
+### Properties
+
+<ResponseField name="iceServers" type="string[]">
+  Array of STUN/TURN server URLs for ICE connection establishment. Default is `["stun:stun.l.google.com:19302"]`.
+
+```javascript
+// Set custom ICE servers
+transport.iceServers = [
+  "stun:stun.l.google.com:19302",
+  "stun:stun1.l.google.com:19302",
+];
+```
+
+</ResponseField>
+
+### Methods
+
+Most methods have an equivalent in the RTVI client. The transport is designed to be used with the RTVI client, so in most cases you should use the RTVI client methods to interact with the transport, rather than calling these methods directly. This is especially true for `initialize()`, `connect()`, and `disconnect()`.
+
+<ResponseField name="setAudioCodec" type="method">
+  Sets the preferred audio codec.
+
+```javascript
+transport.setAudioCodec("opus");
+```
+
+</ResponseField>
+
+<ResponseField name="setVideoCodec" type="method">
+  Sets the preferred video codec.
+
+```javascript
+transport.setVideoCodec("VP8");
+```
+
+</ResponseField>
+
+### States
+
+The transport can be in one of these states:
+
+```typescript
+type TransportState =
+  | "initializing"
+  | "initialized"
+  | "connecting"
+  | "connected"
+  | "ready"
+  | "disconnecting"
+  | "disconnected"
+  | "error";
+```
+
+## Events
+
+The transport implements the various [RTVI event handlers](https://docs.pipecat.ai/client/js/api-reference/callbacks). These events will fire when used with an RTVIClient.
+
+Key callback events:
+
+<ResponseField name="onConnected" type="callback">
+  Called when connection to the server is established and the data channel is
+  ready.
+</ResponseField>
+
+<ResponseField name="onDisconnected" type="callback">
+  Called when connection to the server is lost.
+</ResponseField>
+
+<ResponseField name="onTransportStateChanged" type="callback">
+  Called when the transport state changes.
+</ResponseField>
+
+<ResponseField name="onTrackStarted" type="callback">
+  Called when a new track is received from the server.
+</ResponseField>
+
+## Connection Process
+
+The connection process follows these steps:
+
+1. The client creates a new RTCPeerConnection
+2. Audio and video transceivers are added
+3. A data channel is created for messaging
+4. An offer is created and sent to the server endpoint
+5. The server responds with an SDP answer
+6. The connection is established when ICE candidates are exchanged
```

Comment:
this is essentially describing how webRTC works and feels weird to document. How about a higher-level/more pipecat specific description?

1. The transport negotiates a WebRTC connection with the corresponding pipecat transport, complete with transceivers for the media and a data channel for messaging.
2. The transport sends a message to the pipecat transport to let it know it's ready.
3. The pipecat transport sends a message letting the client know it is ready.